firstvoices/backend/search/indices/dictionary_entry_document.py

```diff
@@ -60,3 +61,43 @@ def delete_from_index(sender, instance, **kwargs):
         logger.warning(
             f"Indexed document not found. Cannot delete from index. document id: {instance.id}"
         )
+
+
+# Translation update
+@receiver(post_save, sender=Translation)
+def update_translation(sender, instance, **kwargs):
+    logger = logging.getLogger(ELASTICSEARCH_LOGGER)
+    dictionary_entry = instance.dictionary_entry
+
+    try:
+        dictionary_entry_doc = DictionaryEntryDocument.get(id=dictionary_entry.id)
+        dictionary_entry_doc.update(translation=instance.text)
+    except ConnectionError:
+        logger.warning(
+            f"Elasticsearch server down. Translation could not be updated in index. document id: {instance.id}"
+        )
+    except NotFoundError:
+        logger.warning(
+            f"Indexed document not found. Cannot updated dictionary_entry with translation. "
+            f"document id: {dictionary_entry.id}"
+        )
+
+
+# Translation update
+@receiver(post_delete, sender=Translation)
+def delete_translation(sender, instance, **kwargs):
+    logger = logging.getLogger(ELASTICSEARCH_LOGGER)
+    dictionary_entry = instance.dictionary_entry
+
+    try:
+        dictionary_entry_doc = DictionaryEntryDocument.get(id=dictionary_entry.id)
+        dictionary_entry_doc.update(translation=None)
+    except ConnectionError:
+        logger.warning(
+            f"Elasticsearch server down. Translation could not be updated in index. document id: {instance.id}"
```

Comment:
These messages are long and repeated. Probably worth putting them in variables, maybe somewhere common for search-related errors